omegaconf/nodes.py

```diff
@@ -335,16 +332,13 @@ def __init__(
             ),
         )
 
-    def validate_and_convert(self, value: Any) -> Optional[Enum]:
+    def _validate_and_convert_impl(self, value: Any) -> Optional[Enum]:
         return self.validate_and_convert_to_enum(enum_type=self.enum_type, value=value)
 
     @staticmethod
     def validate_and_convert_to_enum(
-        enum_type: Type[Enum], value: Any, allow_none: bool = True
+        enum_type: Type[Enum], value: Any
     ) -> Optional[Enum]:
```

Comment:
Return type should be `-> Enum` instead of `-> Optional[Enum]`.

Reply:
Fixed in 8e16ce4